src/app/components/Icons/LocationIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function LocationIcon(): JSX.Element {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="42" height="42">
+      <path
+        d="M572-159a20.87,20.87,0,0,1-8.174-1.65,20.93,20.93,0,0,1-6.675-4.5,20.929,20.929,0,0,1-4.5-6.675A20.868,20.868,0,0,1,551-180a20.868,20.868,0,0,1,1.65-8.174,20.929,20.929,0,0,1,4.5-6.675,20.93,20.93,0,0,1,6.675-4.5A20.871,20.871,0,0,1,572-201a20.868,20.868,0,0,1,8.174,1.65,20.929,20.929,0,0,1,6.675,4.5,20.929,20.929,0,0,1,4.5,6.675A20.868,20.868,0,0,1,593-180a20.868,20.868,0,0,1-1.65,8.174,20.929,20.929,0,0,1-4.5,6.675,20.929,20.929,0,0,1-6.675,4.5A20.868,20.868,0,0,1,572-159Zm-.055-31a6.9,6.9,0,0,0-4.912,2.033A6.9,6.9,0,0,0,565-183.055c0,2.691,1.888,6.055,3.473,8.4a45.335,45.335,0,0,0,3.473,4.5,45.807,45.807,0,0,0,3.473-4.5c1.584-2.348,3.473-5.712,3.473-8.4a6.9,6.9,0,0,0-2.033-4.912A6.9,6.9,0,0,0,571.945-190Zm0,9.427a2.484,2.484,0,0,1-2.48-2.481,2.483,2.483,0,0,1,2.48-2.48,2.483,2.483,0,0,1,2.48,2.48A2.484,2.484,0,0,1,571.945-180.573Z"
+        transform="translate(-551 201)"
+        fill="#269396"
```

Comment:
```suggestion
        fill="currentColor"
```
like on the other icons

Reply:
Thanks! I did not see that.